src/main/scala/zio/zmx/MetricSnapshot.scala

```diff
@@ -1,8 +1,10 @@
 package zio.zmx
 
-sealed trait MetricSnapshot
+sealed trait MetricSnapshot[+T] {
```

Comment:
Don't think there is a need for a type parameter here as this will always be a `String`.